templates/RELEASE_NOTES_unified.htm.tmpl

```diff
@@ -31,8 +31,7 @@
     <p>The following components were introduced or enhanced in the {{ toLower .Version }} Conjur OSS suite.</p>
     <MadCap:listOfProxy style="mc-list-of-tag: h3;mc-list-of-class: list;mc-list-of-paragraph-class: RN;" />
 
-    {{- range .Components }}
-
+    {{- range .Components }}{{ $length := len .Changelogs }}{{ if ne $length 0 }}
```

Comment:
Use multiple lines here and you can use `{{-` for removing extra whitespace - there's no need for inlined stacking